Hangman/Hangman/main.swift

```diff
@@ -2,11 +2,167 @@
 //  main.swift
 //  Hangman
 //
-//  Created by Alex Paul on 10/15/18.
-//  Copyright © 2018 Pursuit. All rights reserved.
+//  Created by Bienbenido Angeles on 10/4/19.
+//  Copyright © 2019 Pursuit. All rights reserved.
 //
 
 import Foundation
+//array of words to choose
```

Comment:
Please remember to remove comments you have left for yourself or blocks of unused code before submitting :)

Reply:
I included them so that I can read through my code more easily at a glance. But I will remove commented out blocks of code that are no long needed